sources/packages/api/scripts/setupDB.ts

```diff
@@ -1,29 +1,23 @@
-require("../env_setup");
-import { createConnection } from "typeorm";
-const config = require("../ormconfig");
+import { ormConfig, simsDataSource } from "../src/database/config/ormconfig";
```

Comment:
I believe that the pattern for the file naming would be `orm-config`.

Reply:
renaming to data-source.ts